hosted/eks/helper/helper_cluster.go

```diff
@@ -316,7 +316,8 @@ func DeleteEKSClusterOnAWS(eks_region string, clusterName string) error {
 	_ = os.Setenv("KUBECONFIG", downstreamKubeconfig)
 
 	fmt.Println("Deleting EKS cluster ...")
-	args := []string{"delete", "cluster", "--region=" + eks_region, "--name=" + clusterName, "--wait", "--force"}
+	// TODO: Fix and wait for cluster deletion
```

Comment:
eksctl should return once the resource is deleted so not sure if there is anything else to do

Reply:
It returns and logs status, however cluster sometimes remains. `--wait` is flaky hence the TODO.
`Running command: eksctl [delete cluster --region=*** --name=auto-eks-hp-ci-ytipy --disable-nodegroup-eviction]
 Deleted EKS cluster:  auto-eks-hp-ci-ytipy
`